src/Test/UnitTests/attributes.jl

```diff
@@ -31,6 +31,27 @@ function silent(model::MOI.ModelLike, config::TestConfig)
 end
 unittests["silent"] = silent
 
+"""
+    timelimitsec(model::MOI.ModelLike, config::TestConfig)
+
+Test that the [`MOI.TimeLimitSec`](@ref) attribute is implemented for `model`.
+"""
+function timelimitsec(model::MOI.ModelLike, config::TestConfig)
+    if config.solve
+        @test MOI.supports(model, MOI.TimeLimitSec())
+        # Get the current value to restore it at the end of the test
+        value = MOI.get(model, MOI.TimeLimitSec())
+        MOI.set(model, MOI.TimeLimitSec(), 0.0)
+        @test isapprox(MOI.get(model, MOI.TimeLimitSec()), 0.0, atol = eps(0.0))
```

Comment:
If you set the time limit to `0.0`, `get` should return `0.0`. There's no need for `isapprox`.